assets/components/contour/Chart.yaml

```diff
@@ -0,0 +1,11 @@
+apiVersion: v2
+name: contour
+version: v1.3.0
+appVersion: v1.3.0
+description: Contour is a Kubernetes ingress controller using Lyft's Envoy proxy.
+home: https://projectcontour.io/
+icon: https://github.com/projectcontour/contour/raw/master/contour.png
+sources:
+  - https://github.com/projectcontour/contour
+engine: gotpl
+
```

Comment:
I would suggest to use what `helm create` generates as a base + optional fields:
```suggestion
apiVersion: v2
name: contour
description: A Helm chart for Contour

# A chart can be either an 'application' or a 'library' chart.
#
# Application charts are a collection of templates that can be packaged into versioned archives
# to be deployed.
#
# Library charts provide useful utilities or functions for the chart developer. They're included as
# a dependency of application charts to inject those utilities and functions into the rendering
# pipeline. Library charts do not define any templates and therefore cannot be deployed.
type: application

# This is the chart version. This version number should be incremented each time you make changes
# to the chart and its templates, including the app version.
version: 0.1.0

# This is the version number of the application being deployed. This version number should be
# incremented each time you make changes to the application.
appVersion: 1.3.0

home: https://projectcontour.io/
icon: https://github.com/projectcontour/contour/raw/master/contour.png
sources:
  - https://github.com/projectcontour/contour
```

Reply:
done